src/styles/_global.scss

```
@@ -1,5 +1,13 @@
+@font-face {
+  font-family: "Brockmann";
```

Stylelint (17.14.1)

[error] Expected no quotes around "Brockmann" (font-family-name-quotes)

(font-family-name-quotes)